### DISCUSSION TITLE
How to prevent focus of first item when <p-menu [popup]="true">

### DISCUSSION THREAD

**Discussion by timothyTheCoder** [primefaces/community#646] [PrimeNG] [answered]
```
<p-menu #menu [model]="items" [popup]="true" appendTo="body"></p-menu>
<button (click)="menu.toggle($event)">Menu</button>
```

When user clicks the button to open the menu, the first element is focused with styling but how to NOT do that and have nothing focused (and thus also not styled)?
As a reference see how menu items work om MacOS. When you open a menu, there is no auto focus of the first element. Only when you click arrow DOWN will the first menu item be focussed.

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

It's made according to accessibility. Thanks for the feedback, we've added it in our internal backlog.
